framework/OutStreams/OutStreamPlot.py

```diff
@@ -1772,12 +1772,54 @@ def addOutput(self):
                   for k, col in zip(range(int(clusterDict[pltindex]['noClusters'])), colors):
                     myMembers = self.clusterValues[pltindex][1][0] == k
                     self.actPlot = plt.scatter(clusterDict[pltindex]['clusterValues'][myMembers, 0], clusterDict[pltindex]['clusterValues'][myMembers, 1] , color = col, **dataMiningPlotOptions)
+
+                  ## Handle all of the outlying data
+                  myMembers = self.clusterValues[pltindex][1][0] == -1
+                  ## resize the points
+                  dataMiningPlotOptions['s'] /= 2
```

Comment:
's' is coming from matplotlib's [scatter](https://matplotlib.org/api/_as_gen/matplotlib.axes.Axes.scatter.html#matplotlib.axes.Axes.scatter). It represents the size of the marker. Here I am reducing the size of the marker to render the "outliers" (the points with labels < 0 as determined by the sklearn algorithms).